docs/src/selectors.md

```diff
@@ -1,29 +1,40 @@
-# Selectors
+# Selectors and LookupArrays
 
-Indexing by value in `DimensionalData` is done with [Selectors](@ref).
-IntervalSets.jl is now used for selecting ranges of values (formerly `Between`).
+http://localhost:5173/DimensionalData.jl/reference#lookuparrays
```

Comment:
Is this going to work with a localhost link? Or should this be replaced by a proper link once the docs are updated?